examples/sdk-examples/src/examples/chartResponsiveness/index.tsx

```diff
@@ -28,12 +37,25 @@ export const ChartResponsiveness: React.FC = () => (
         <h2 id="bar-chart">Visual Components</h2>
         <p>
             Small visual components will have parts of the chart hiden to improve user responsiveness
-            experience.
+            experience. Various widget heights can be seen in this example. Smallest has only the axis title
+            visible, middle has only the axis labels visible and the largest has both axis title and labels
+            visible.
```

Comment:
When rendered in a smaller size, visual components hide different sections to improve a visual user experience. You can see how the chart changes when you switch between different heights. With the smallest value, the chart has only the axis title visible. With the middle value, the chart has only the axis labels visible. With the largest value, the chart has both the axis title and labels visible.